### ISSUE TITLE
Fix changed default size of the touchscreen formspec

### ISSUE THREAD

**Issue by S-S-X** [mt-mods/digistuff#32]
> The default size of the formspec changed from width 13 and height 10 to width 10 and height 8

I did some testing, and this is actually a bug with the old touchscreens when real coordinates are enabled. They have always been 10 by 8, but because the `real_coordinates` element was not placed immediately after the `size` element, it caused the formspec to be bigger than it was supposed to be.

See here, the size before and after the rewrite is the same:
https://github.com/mt-mods/digistuff/blob/b2369b3e815bc986bfdac8e4d9015c16129c5a6d/touchscreen.lua#L3
https://github.com/mt-mods/digistuff/blob/a49e4ddeaf4e693d76d4d47d0382d46c654f3d2d/touchscreen.lua#L121-L122

_Originally posted by @OgelGames in https://github.com/mt-mods/digistuff/issues/25#issuecomment-1021915930_

This should be fixed (it is irrelevant if it was "bug" in original, behavior can probably be restored without reintroducing invalid use of real_coordinates).

**Comment by OgelGames** [comment]
I don't think this should be "fixed", as doing so is effectively reintroducing the bug. Formspecs should remain the same size whether real coordinates are used or not. (and I mean the set size, not the visual size)

Also there is a simple fix for it:
> Can be fixed ingame by adding: {command = "set", width = 13, height = 10}

It might be good to mention it in the documentation though.

**Comment by S-S-X** [comment]
> as doing so is effectively reintroducing the bug

I've probably misunderstood original issue, how exactly changing default size to follow original behavior reintroduces bug?
